### PR TITLE
Dockerize app

### PR DESCRIPTION
**Pull request by johnnymo87**
Pairing with @nnarh on this.

TODO:
* [ ] Dockerize heroku deployments
* [ ] Dockerize tests in CI

These remaining tasks could be separate PRs.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -45,10 +45,15 @@
 end
 
 Logger.new(STDOUT).info 'Start features seed'
+Feature.create(name: 'volunteer_ops_create')
```
Comment (tansaku) [inline db/seeds.rb]: it makes sense for these create things to here - I wonder if we removed them for a purpose.  I know we concluded that we should avoid data being generated in migrations which was where they were originally ...

**Comment by tansaku** [comment]
great start @johnnymo87 and @nnarh - working through the docs now

File: docs/installation/Developing-With-Docker.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+```
+
+This will give you output similar to:
+
+```
+NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER    ERRORS
+default   *        virtualbox   Running   tcp://192.168.99.100:2376           v1.11.2
+```
+
+## Configuring namely.lvh.me to point to VirtualBox
```
Comment (tansaku) [inline docs/installation/Developing-With-Docker.md]: should this still say namely.lvh.me?

**Comment by tansaku** [comment]
was working through the install:

`docker-compose build` was going along happily until:

```
Fetched 81.2 MB in 14s (5549 kB/s)
E: Failed to fetch http://httpredir.debian.org/debian/pool/main/n/node-retry/node-retry_0.6.0-1_all.deb  Error reading from server. Remote end closed connection [IP: 5.153.231.35 80]

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
ERROR: Service 'app' failed to build: The command '/bin/sh -c apt-get update &&     apt-get -y install qt5-default                        libqt5webkit5-dev                        gstreamer1.0-plugins-base                        gstreamer1.0-tools                        gstreamer1.0-x                        xvfb                        npm                        nodejs-legacy' returned a non-zero code: 100
```

**Comment by johnnymo87** [comment]
@tansaku Try commenting out that line and everything below, and replace it with `CMD bash`. Then `docker-compose build` and `docker-compose run app bash` to drop into a container. Then try the apt-get line?

**Comment by tansaku** [comment]
will do @johnnymo87

**Comment by johnnymo87** [comment]
Also, is there any chance you lost network connectivity during this step?

**Comment by tansaku** [comment]
@johnnymo87 one thing we should watch here is that the branch/PR links to a ticket in the tracker - even if it's just a placeholder to say what docker is for and why it's a good idea etc.

**Pull request merged**